src/EntityId/EntityIdFormatter.php

```diff
@@ -17,6 +17,7 @@ interface EntityIdFormatter {
 	 * @param EntityId $value
 	 *
 	 * @return string Either plain text, wikitext or HTML.
+	 * @throws \IllegalArgumentException when value falls out of supported patterns by the formatter
```

Comment:
Not on a dev machine so I can't check - should this not be InvalidArgumentException? That is what the phab ticket says

Reply:
yes my bad .. fixed now